bot/handlers/administration/remove_whitelist_handler.py

```diff
@@ -37,12 +39,27 @@ async def __check_user_id_digit(self, message: Message) -> bool:
             return False
         return True
 
+    async def __check_user_exists(self, message: Message) -> bool:
+        user_id = int(message.text.split()[1])
+        user_exists = await DatabaseManager.is_user_in_db(user_id)
+        if not user_exists:
+            await self._reply_user_not_found(message, user_id)
+            return False
+        return True
+
     async def _do_handle(self, message: Message) -> None:
         user_id = int(message.text.split()[1])
 
         await DatabaseManager.remove_user(user_id)
         await self.__reply_user_removed(message, user_id)
 
     async def __reply_user_removed(self, message: Message, user_id: int) -> None:
-        await self._answer(message,get_user_removed_message(str(user_id)))
-        await self._log_system_message(logging.INFO, get_log_user_removed_message(str(user_id), message.from_user.username))
+        await self._answer(message, get_user_removed_message(str(user_id)))
+        await self._log_system_message(
+            logging.INFO,
+            get_log_user_removed_message(str(user_id), message.from_user.username),
+        )
+
+    async def _reply_user_not_found(self, message: Message, user_id: int) -> None:
+        await self._answer(message, get_user_not_in_whitelist_message(user_id))
+        await self._log_system_message(logging.WARNING, f"User with ID {user_id} not found.")
```

Comment:
//optional w sumie sam nie wiem ale chyba miales funkcje na te logi a nie takie raw stringi wrzucac, czy cos pomieszalem? zmien albo resolve, whatever xD

Reply:
A to na RAW string będzie funkcja to przeoczyłem.